### ISSUE TITLE
cannot toggle filters using the keyboard in many cases

### ISSUE THREAD

**Issue by gdementen** [larray-project/larray-editor#70]
Unsure what the condition for this is. I **guess**, it is because the filter widget does not receive (ask for?) the keyboard focus.

**Comment by alixdamman** [comment]
"in many cases" is very vague. Need to see it.

**Comment by gdementen** [comment]
I know... I don't know the exact conditions it happens.
